docs/src/_guide/orchestration.md

```diff
@@ -13,6 +13,7 @@ When a new pipeline schedule is created using the [UI](/reference/ui) or [CLI](/
 
 ## Create a Schedule
 
+### Specifying extractors and loaders
 To regularly schedule your ELT to run, use the ["Pipelines" interface in the UI](/reference/ui#pipelines), or the [`meltano schedule`  command](/reference/command-line-interface#schedule):
```

Comment:
Sorry, total nit here but my markdown linter will always complain if there isn't a line between header and content.

```suggestion
### Specifying extractors and loaders

To regularly schedule your ELT to run, use the ["Pipelines" interface in the UI](/reference/ui#pipelines), or the [`meltano schedule`  command](/reference/command-line-interface#schedule):
```

What do you think of a blank line here and elsewhere as applicable?

Reply:
Good callout, I apparently had disabled running `prettier` on file save in my editor. I just ran these through `prettier` now and it added all the appropriate whitespace.